common/modules/olm/manifests/1.4.0/amq-streams.v1.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1214 @@
+                - /opt/strimzi/bin/cluster_operator_run.sh
+                env:
+                - name: STRIMZI_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: STRIMZI_FULL_RECONCILIATION_INTERVAL_MS
+                  value: "120000"
+                - name: STRIMZI_OPERATION_TIMEOUT_MS
+                  value: "300000"
+                - name: STRIMZI_DEFAULT_ZOOKEEPER_IMAGE
+                  value: registry.redhat.io/amq7/amq-streams-kafka-24-rhel7:1.4.0
+                - name: STRIMZI_DEFAULT_TLS_SIDECAR_ENTITY_OPERATOR_IMAGE
+                  value: registry.redhat.io/amq7/amq-streams-kafka-24-rhel7:1.4.0
+                - name: STRIMZI_DEFAULT_TLS_SIDECAR_KAFKA_IMAGE
+                  value: registry.redhat.io/amq7/amq-streams-kafka-24-rhel7:1.4.0
+                - name: STRIMZI_DEFAULT_TLS_SIDECAR_ZOOKEEPER_IMAGE
+                  value: registry.redhat.io/amq7/amq-streams-kafka-24-rhel7:1.4.0
+                - name: STRIMZI_DEFAULT_KAFKA_EXPORTER_IMAGE
+                  value: registry.redhat.io/amq7/amq-streams-kafka-24-rhel7:1.4.0
+                - name: STRIMZI_KAFKA_IMAGES
+                  value: |
+                    2.3.0=registry.redhat.io/amq7/amq-streams-kafka-23-rhel7:1.4.0
+                    2.4.0=registry.redhat.io/amq7/amq-streams-kafka-24-rhel7:1.4.0
+                - name: STRIMZI_KAFKA_CONNECT_IMAGES
+                  value: |
+                    2.3.0=registry.redhat.io/amq7/amq-streams-kafka-23-rhel7:1.4.0
+                    2.4.0=registry.redhat.io/amq7/amq-streams-kafka-24-rhel7:1.4.0
+                - name: STRIMZI_KAFKA_CONNECT_S2I_IMAGES
+                  value: |
+                    2.3.0=registry.redhat.io/amq7/amq-streams-kafka-23-rhel7:1.4.0
+                    2.4.0=registry.redhat.io/amq7/amq-streams-kafka-24-rhel7:1.4.0
+                - name: STRIMZI_KAFKA_MIRROR_MAKER_IMAGES
+                  value: |
+                    2.3.0=registry.redhat.io/amq7/amq-streams-kafka-23-rhel7:1.4.0
+                    2.4.0=registry.redhat.io/amq7/amq-streams-kafka-24-rhel7:1.4.0
+                - name: STRIMZI_KAFKA_MIRROR_MAKER_2_IMAGES
+                  value: |
+                    2.4.0=registry.redhat.io/amq7/amq-streams-kafka-24-rhel7:1.4.0
+                - name: STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE
+                  value: registry.redhat.io/amq7/amq-streams-rhel7-operator:1.4.0
+                - name: STRIMZI_DEFAULT_USER_OPERATOR_IMAGE
+                  value: registry.redhat.io/amq7/amq-streams-rhel7-operator:1.4.0
+                - name: STRIMZI_DEFAULT_KAFKA_INIT_IMAGE
+                  value: registry.redhat.io/amq7/amq-streams-rhel7-operator:1.4.0
+                - name: STRIMZI_DEFAULT_KAFKA_BRIDGE_IMAGE
+                  value: registry.redhat.io/amq7/amq-streams-rhel7-operator:1.4.0
+                - name: STRIMZI_CREATE_CLUSTER_ROLES
```

Comment:
Thsi is wrong IMHO:
* You ask it to create the cluster roles, but you didn't give it the rights to do it.
* For OSD, we replaced this with having the ClusterRoles as YAMLs in the directory -> they seem to be missing.

I think that the RBAC is ok, you should remove this and add the cluster role files.

Reply:
Yeah, you are right. Just added.